README.md

```diff
@@ -37,37 +37,37 @@ To use this in your application, simply add this in your pom.xml:
 
 ```xml
     <dependency>
-        <groupId>io.quarkiverse.loggingui</groupId>
-        <artifactId>quarkus-logging-ui</artifactId>
-        <version>${logger-ui.version}</version>
+        <groupId>io.quarkiverse.loggingmanager</groupId>
+        <artifactId>quarkus-logging-manager</artifactId>
```

Comment:
While you're at it, we will request soon all the Quarkiverse extensions to have their name starting with `quarkiverse-` so that should be `quarkiverse-logging-manager`. Better do it in one go :).

Reply:
done

Reply:
Hmmmm... apparently... it's not sold yet. Discussions are still ongoing.

Keep the commit separate for now, I'll let you know how it goes.

Reply:
I will have to undo this commit then.
This change makes sense, I wonder why it is not sold yet.

Reply:
It's hum... complicated :).

I'll let you know here as soon as the final decision is taken.

Reply:
Seems like this is correct now ? See  https://github.com/quarkiverse/quarkiverse/pull/23/files is

Reply:
Yes, this is correct. The repository name has changed though, I added a comment for that.